modules/gdscript/gdscript_compiler.h

```diff
@@ -139,6 +139,7 @@ class GDScriptCompiler {
 	bool within_await = false;
 
 public:
+	static void _initializer_implicit_conversion(Variant &p_variant, const GDScriptParser::VariableNode *p_node);
```

Comment:
Not sure about the method's name. Should me more something like `_implicitly_convert_initializer()` or something like that.

Reply:
Good call! I ended up removing the mention of implicit conversion.... since all this does is convert. The implicit conversion happens when this is called :)